### ISSUE TITLE
[activityreminder] Uncaught TypeError: Assignment to a constant

### ISSUE THREAD

**Issue by myxor** [espruino/BangleApps#1949]
### Affected hardware version

Bangle 2

### Your firmware version

2v13

### The bug

Seeing 

```
Uncaught TypeError: Assignment to a constant
 at line 4 col 45 in activityreminder.app.js
const activityreminder=require("activityreminder");
```

in the log when starting the app via launcher and the app does not start.

----

I did not have time to look into details why and where this happens.

Ping @Stiralbios

**Comment by Stiralbios** [comment]
Thanks,

I don't have the exception. However I think it's because I use this const in the boot.js and app.js. I read that recently espruino firmware started handling the const, let, etc... properly. Are you on the cutting edge firmware ?if so it's maybe why you see it and I don't.
I will try to fix it tomorrow.

**Comment by myxor** [comment]
Yes i am on 2v13.158 cutting edge build.

11.06.2022 18:11:55 Stiralbios ***@***.***>:

**Comment by myxor** [comment]
Today i saw a different exception happening:

```
Uncaught ReferenceError: "activityreminder_data" is not defined
 at line 26 col 49 in .boot0
...eps - activityreminder_data.stepsOnDate >= activityreminder_...
                              ^
in function called from system
```

**Comment by Stiralbios** [comment]
This one is odd, not quite sure if it come from the app or, maybe, the firmware. I don't see any way for to arrive in this part of the code without this const being defined.

**Comment by gfwilliams** [comment]
fixed with #1951
